### DISCUSSION TITLE
AlDente 1.28.2

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1313] [General]
### All

- Fixed some UI issues in the new settings window

### All

- New App Icon!
- New Settings window which can better visualize AlDente's features and settings, especially on smaller screens. 
- New and improved popover window with tweaked design to better fit the macOS visual language.
- Removed the "Popup Animation" option due to the new popover window.
- Many smaller changes and fixes.

### Pro

- Users of Sailing Mode can now charge back to the charge limit by double-clicking the slider in the popup window.
- A warning icon in AlDente's status display is added if the app detects that the power supply is plugged in but cannot provide power.
- Added the ability to control the color of the Magsafe LED for MacBooks with MagSafe 3 via the Apple Shortcuts app.
- Added the ability to turn the Magsafe LED off while the device is asleep.

With peace & love from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.28.2'>AlDente 1.28.2</a>.</em>

**Comment by fingers986** [comment]
With 1.28.2, can I set LED to orange when battery is charging, and turn off LED when hold or discharge?

**Reply by MatthiasKerbl** [reply]
Hi @fingers986,

The Magsafe LED control was now added to Apple Shortcuts and can therefore be configured manually the way you want it. Directly within the AlDente settings, your scenario is not possible.

Best,
Matthias
